From ad49bfcb6a29dda93170ea14c38d599df2bbf72d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?R=C3=A9mi=20Cardona?= <remi@gentoo.org>
Date: Sun, 14 Feb 2010 21:39:50 +0100
Subject: [PATCH 1/2] properly detect gmodule

This usually doesn't happen unless gcc is configured to force
--as-needed through spec files.
---
 configure.in |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/configure.in b/configure.in
index 972fbc2..e2e84ce 100644
--- a/configure.in
+++ b/configure.in
@@ -32,7 +32,7 @@ AC_CHECK_LIB(intl,main)
 dnl
 dnl Standard GTK checks
 dnl
-PKG_CHECK_MODULES(GTK, gtk+-2.0 >= 2.2.0 glib-2.0 >= 2.2)
+PKG_CHECK_MODULES(GTK, [gtk+-2.0 >= 2.2.0 glib-2.0 >= 2.2 gmodule-2.0 >= 2.2])
 LDFLAGS="${LDFLAGS} ${GTK_LIBS}"
 CFLAGS="${CFLAGS} ${GTK_CFLAGS}"
 
@@ -44,8 +44,8 @@ dnl
 AC_MSG_CHECKING(whether GLib supports loadable modules)
 ac_save_CFLAGS="$CFLAGS"
 ac_save_LIBS="$LIBS"
-CFLAGS="$CFLAGS $GLIB_CFLAGS"
-LIBS="$GLIB_LIBS $LIBS"
+CFLAGS="$CFLAGS $GTK_CFLAGS"
+LIBS="$GTK_LIBS $LIBS"
 AC_TRY_RUN([
 #include <glib.h>
 #include <gmodule.h>
-- 
1.6.6.1

